airframe-sql/src/test/resources/wvlet/airframe/sql/standard/queries.yml

```diff
@@ -224,3 +224,5 @@
       select EXTRACT(TIMEZONE_HOUR FROM FROM_UNIXTIME(time)), COUNT(*) FROM a
 - sql: |
       select EXTRACT(timezone_minute FROM FROM_UNIXTIME(time)), COUNT(*) FROM a
+- sql: |
+      select user_agent || 'x', count(*) from impression group by 1
```

Comment:
In case we need to preserve this original syntax, it's better to define a new Expression class like `case class Concatenation(values: Seq[Values]) extends FunctionCall(...)` and add a rule for this in SQLGenerator.